docs/sphinx/source/installation.rst

```diff
@@ -34,49 +33,153 @@ There are many ways to install Python on your system, but the Anaconda
+more details.
 
+.. _setupenvironment:
 
-Working at the bleeding edge
-----------------------------
+Set up a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We strongly recommend working in a `virtual environment
 <http://astropy.readthedocs.org/en/latest/development/workflow/
-virtual_pythons.html>`_ if you're going to use the development versions
-of the code. There are many ways to use virtual environments in Python,
-but Anaconda again provides the easiest solution:
+virtual_pythons.html>`_ if you're going to use an editable version
+of the library. You can skip this step if:
 
-#. Create a new conda environment for pvlib and pre-install a
-   handful of packages into the environment:
-   ``conda create --name pvlibdev python pandas scipy``
-#. Activate the new environment: ``source activate pvlibdev``
-#. Install the latest development version of pvlib:
+#. You already have Anaconda or another scientific Python distribution
+#. You don't mind polluting your Python installation with your
+   development version of pvlib.
+#. You don't want to work with multiple versions of pvlib.
 
-  #. If you don't plan to modify the source-code:
-     ``pip install git+https://github.com/pvlib/pvlib-python.git``
-  #. If you do plan to modify the source code:
-     Use the GitHub GUI application or git command-line tool to
-     clone this repository to your computer, then navigate your
-     command-line to the top-level pvlib-python directory,
-     then ``pip install -e .``
+There are many ways to use virtual environments in Python,
+but Anaconda again provides the easiest solution. These are often
+referred to as *conda environments*, but they're the same for our purposes.
 
-#. You may also consider installing additional packages into your
-   development environment:
-   ``conda install jupyter ipython seaborn nose flake8``
+#. **Create** a new conda environment for pvlib and pre-install
+   the required packages into the environment:
+   ``conda create --name pvlibdev python pandas scipy``
+#. **Activate** the new conda environment: ``source activate pvlibdev``
+#. **Install** additional packages into your development environment:
+   ``conda install jupyter ipython matplotlib seaborn nose flake8``
 
 The `conda documentation
 <http://conda.pydata.org/docs/using/index.html>`_ has more information
-on how to use virtual environments. You can also add ``-h`` to most
-conda commands to get help (e.g. ``conda -h`` or ``conda env -h``)
+on how to use conda virtual environments. You can also add ``-h`` to most
+pip and conda commands to get help (e.g. ``conda -h`` or ``conda env -h``)
+
+.. _installsource:
+
+Install the source code
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Good news -- installing the source code is the easiest part!
+
+#. In your terminal or shell, **navigate** to the top-level
+   pvlib-python directory.
+#. **Install** pvlib-python in "development mode" by running
+   ``pip install -e .``
+
```

Comment:
I think this is not applicable as written.

If one uses github 4 windows, the clone will be in `C:\Users\%USER%\Documents\GitHub\pvlib-python`.

So the user must have

1. the environment activated
1. `pip install -e C:\Users\%USER%\Documents\GitHub\pvlib-python` from within the environment.